Demo/Common/IntegrationExamples/Components/BLIKComponentAdvancedFlowExample.swift

```diff
@@ -0,0 +1,145 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenActions
+import AdyenCheckout
+import AdyenComponents
+
+internal final class BLIKComponentAdvancedFlowExample: InitialDataAdvancedFlowProtocol {
+
+    internal weak var presenter: PresenterExampleProtocol?
+    
+    private var adyenCheckout: AdyenCheckout?
+    private var component: AdyenCheckoutComponent?
+    
+    internal lazy var apiClient = ApiClientHelper.generateApiClient()
+    
+    // comes from demo app protocol, unused on new structure
+    internal lazy var context: AdyenContext = generateContext()
+    
+    internal init() {}
+    
+    internal func start() {
+        presenter?.showLoadingIndicator()
+        requestPaymentMethods(order: nil) { [weak self] result in
+            guard let self else { return }
+            
+            do {
+                switch result {
+                case let .success(paymentMethods):
+                    try self.presentBlik(from: paymentMethods)
+                case let .failure(error):
+                    throw error
+                }
+            } catch {
+                self.presenter?.hideLoadingIndicator()
+                self.presenter?.presentAlert(withTitle: "Error", message: error.localizedDescription)
+            }
+        }
+    }
+    
+    private func presentBlik(from paymentMethods: PaymentMethods) throws {
+        guard let blikPaymentMethod = paymentMethods.paymentMethod(ofType: BLIKPaymentMethod.self) else { throw IntegrationError.paymentMethodNotAvailable(paymentMethod: BLIKPaymentMethod.self) }
+        
+        let configuration = try CheckoutConfiguration(
+            environment: ConfigurationConstants.componentsEnvironment,
+            amount: ConfigurationConstants.current.amount,
+            clientKey: ConfigurationConstants.clientKey,
+            analyticsConfiguration: .init(
+                isEnabled: ConfigurationConstants.current.analyticsSettings.isEnabled
```

Comment:
Confirming my assumption from above - `isEnabled` is a bit ambiguous now

Reply:
I'm not sure on this one, to me `isEnabled` is clear enough since its a property of the configuration that represents analytics and init directly uses the property.